src/content/doc-sdk-golang/methods/kill.mdx

```diff
@@ -40,5 +40,23 @@ db.Kill(queryUuid string)
 ### Example usage
 
 ```go
-await db.Kill(queryUuid);
-```
+package main
+
+import (
+    "fmt"
+    "github.com/surrealdb/surrealdb.go"
+)
+
+func main() {
+    // Assuming db is an instance of your database connection
+    // and uuid is a string containing the query UUID you want to kill
+    queryUuid := "your-query-uuid-here"
+
+    err := db.Kill(queryUuid)
```

Comment:
I looked at the package documentation at https://pkg.go.dev/github.com/surrealdb/surrealdb.go. There the `Kill` function is not on the `DB` type but a stand-alone function taking the DB as a parameter:

```
func Kill(db *DB, id string) error
```

So something like the following should be used:

```suggestion
package main

import (
    "fmt"
    surreal "github.com/surrealdb/surrealdb.go"
)

func main() {
    // Assuming db is an instance of your database connection
    // and uuid is a string containing the query UUID you want to kill
    queryUuid := "your-query-uuid-here"

    err := surreal.Kill(db, queryUuid)
    <remainder same as before>
```

Note that the import has an alias now to make it easier to use.